server/sso/authorization_response.go

```diff
@@ -127,31 +131,57 @@ func (r resp) status() (int, error) {
 	return AuthnFailed, errors.New("malformed or missing auth response")
 }
 
+// RequestID partially implements the fleet.Auth interface.
 func (r resp) RequestID() string {
 	if r.response != nil {
 		return r.response.InResponseTo
 	}
 	return ""
 }
 
-func (r resp) rawResponse() string {
+// AssertionAttributes partially implements the fleet.Auth interface.
+func (r resp) AssertionAttributes() []fleet.SAMLAttribute {
```

Comment:
Sorry, I can't understand what this function is doing.
Looks like you are extracting the attributes from `r.response.Assertion.AttributeStatement.Attributes`, putting it in another attributes var and returning it.

Why?
Maybe rename the function to something meaningful or explain in the comment?

thanks 🙏

Reply:
`AssertionAttributes` returns the attributes present in the assertion (the documentation of what it should do is in the interface that this struct is implementing: https://github.com/fleetdm/fleet/blob/c83794bc6c3eee7e7bafdeed95f7cc3dc2ce637b/server/fleet/sessions.go#L29).

Let me know if it's still not clear.

Reply:
Got it. THX